### DISCUSSION TITLE
Weird Mask

### DISCUSSION THREAD

**Discussion by TheOriginalCodiac** [qwbarch/mirage#125] [Q&A] [answered]
Sometimes, the Masked have these weird green masks on, along with suits, kinda looks alien. And they say some odd noises, is this part of your mod, or some other mod? And if so, is there a way to remove it? Kinda makes it easy to spot them.

Love your mod.

**Comment by qwbarch** [comment]
It's not from Mirage. This mod gives an option to remove masks, but other tham that, it doesn't change anything relating to cosmetics.

If it says weird noises, it _could_ be a Mirage bug, but I'd recommend checking your recordings folder to verify if the audio sounds fine or not (at ``<LC Folder>/Mirage/Recording``)

**Reply by TheOriginalCodiac** [reply]
Do you know of any other mods that might add it? It's not like a voice bug, it's of different voice lines. Custom ones. Not anything either me, or my friend has said. So yeah, to sum it up, it uses a weird/funny looking mask and custom voice lines. Do you have any idea of what mod might do that?

**Reply by qwbarch** [reply]
I haven't really played LC for like the past 8 months so I'm afraid I have no idea what other mods do right now, sorry 😅

**Reply by TheOriginalCodiac** [reply, marked as answer]
Figured it out. It was the Zombies Plushies Mod that added in the 'Zombie' varient of the masked. It doesn't really play well with Mirage. But preventing them from spawning worked well.

**Reply by qwbarch** [reply]
Glad you found the issue, I'll talk to zehs about it
